msp/msp_exp_handler.c

```diff
@@ -621,11 +621,21 @@ void msp_exprecv_complete(unsigned char opcode)
 			citiroc_send_slow_control();
 			break;
 
-		case MSP_OP_SEND_CUBES_PROB_CONF:
-			mem_ram_write(RAM_CITI_PROBE, recv_data);
-			citiroc_send_probes();
+		case MSP_OP_SEND_NVM_CITI_CONF:
+			mem_nvm_write(NVM_CITIROC, recv_data);
 			break;
 
+		case MSP_OP_APPLY_CUBES_CITI_CONF:
+			uint8_t *nvm_cfg_addr=(uint8_t *)(NVM_ADDR+CITIROC_OFFSET+(recv_data[0]*CITIROC_LEN));
```

Comment:
This line causes a compilation error for me, because you are trying to declare a variable within a case statement. You can either add braces around the block of the case statement, or declare the `nvm_cfg_addr` pointer outside the switch statement and just assign the pointer here.

Also, please keep this line within 80 character columns.